src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java

```diff
@@ -139,6 +139,15 @@ public static boolean equals(PersonName personName, String string) {
         return StringUtils.equalsIgnoreCase(normalise(personName.toString()),normalise(string));
     }
 
+    public static boolean equalsIndividualMO(PersonName personName, String string) {
+        var nullValuesCheck = handleNulls(personName, string);
+        if (nullValuesCheck != null) {
+            return nullValuesCheck;
+        }
+        String individualMoFormat = personName.getSurname().concat(", ").concat(personName.getForename());
```

Comment:
Does the UI include a space? We normalise() but that would still be ", "

Reply:
seems its done in https://github.com/companieshouse/api.ch.gov.uk/blob/40479583de693e536c3a05bd50d54e2e8920965b/lib/ApiChGovUk/Controller/CompanyOfficers.pm#L381C8-L384C1

 # Only join surname and forenames if the officer has forenames
        $officer->{name} = join ', ', ($officer->{name}, join ' ', map { $appts_officer->{$_} } @forenames)
            if scalar @forenames;

Reply:
should be ", "